client_test.go

```diff
@@ -31,8 +31,11 @@ import (
 	"github.com/pkg/errors"
 	"github.com/stretchr/testify/assert"
 
+	"fmt"
+
 	dosaRenamed "github.com/uber-go/dosa"
-	"github.com/uber-go/dosa/connectors/devnull"
+	_ "github.com/uber-go/dosa/connectors/devnull"
```

Comment:
Apparently there is an `init()` function in the devnull package. I missed that. woops